src/spec/verifiable-payid-protocol-NC.md

```diff
@@ -0,0 +1,379 @@
+---
+coding: utf-8
+
+title: Verifiable PayID Protocol for Non-Custodial Services
+docname: draft-aanchal-verifiable-payid-protocol-ext-01
+category: std
+
+pi: [toc, sortrefs, symrefs, comments]
+smart_quotes: off
+
+area: security
+author:
+
+  -
+    ins: A. Malhotra
+    name: Aanchal Malhotra
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: amalhotra@ripple.com
+    uri: https://www.ripple.com
+
+  -
+    ins: D. Schwartz
+    name: David Schwartz
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: david@ripple.com
+    uri: https://www.ripple.com
+
+normative:
+    RFC2119:
+    RFC2818:
+    RFC5280:
+    RFC6979:
+    RFC7515:
```

Comment:
Depending on the resolutions of my comments at the bottom of this document, we _may_ be missing RFC-7519 (JWT). 

Also, references should be sorted in ascending order (from the top to the bottom). E.g., 

```
...
RFC7515:
RFC7517:
RFC7519:
...
```

Reply:
Done.